certified-connectors/Progressus Advanced Projects/Info.md/info.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
@ProgressusGlobal We do not require any other .md files in your connector folder in the open source repo. You can provide all the information in the readme.md file.